build/uninstaller/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.16 as builder
+ENV PATH /go/bin:/usr/local/go/bin:$PATH
+ENV GOPATH /go
+COPY . /go/src/github.com/liqotech/liqo
+WORKDIR /go/src/github.com/liqotech/liqo
+
+ARG BUILD_TAGS=""
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=$(go env GOARCH) go build ./cmd/uninstaller/
+RUN cp uninstaller /usr/bin/uninstaller
+
+FROM alpine:3.13.2
```

Comment:
Specifying only 3.13 ensures that you are up to date. 3.13.2 is already out of date

Reply:
I personally prefer to have a slightly outdated version but know precisely which image we are using and have all Liqo components aligned to the same one.